yoti_python_sdk/tests/test_document_details.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+
+from yoti_python_sdk.document_details import DocumentDetails
+
+import datetime
+
+
+def test_exception_for_empty_data():
+    DATA = ""
+
+    try:
+        DocumentDetails(DATA)
+    except ValueError:
+        return
+    assert False  # An exception should have been thrown
+
```

Comment:
Would be a bit neater if we did it this way:
```python
 with pytest.raises(ValueError) as exc:
        DocumentDetails(DATA)
```
and then also asserted that the error message includes what we expect:
```python
    assert str(exc.value) == "Invalid value for DocumentDetails"
```